cypress/e2e/AddingEmployees.ts

```diff
@@ -0,0 +1,59 @@
+import AddEmployeePage from "../pages/AddEmployeePage";
+import DashBoardPage from "../pages/DashBoardPage";
+import LoginPage from "../pages/LoginPage";
+import PIMPage from "../pages/PIMPage";
+import EmployeeListPage from "../pages/EmployeeListPage";
+import PersonalDetailsPage from "../pages/PersonalDetailsPage";
+
+describe("Adding New Employee Functionality", () => {
+  describe("Add New Employee With Admin User", () => {
+    beforeEach(() => {
+      LoginPage.navigateTo(LoginPage.path);
+      LoginPage.login(Cypress.env("username"), Cypress.env("password"));
+    });
+
+    it("As admin I should add a new employee succussfuly", () => {
+      LoginPage.navigateTo(DashBoardPage.path);
+      DashBoardPage.clickElement(DashBoardPage.PIMTab);
+      PIMPage.clickElement(PIMPage.AddEmployeeTab);
+      AddEmployeePage.getElement(AddEmployeePage.employeeIdInput)
+        .invoke("val")
+        .then((id) => cy.task("setEmployeeId", id));
+      cy.task("setEmployee", {
+        username: AddEmployeePage.getRandomUserName(),
+        password: AddEmployeePage.getRandomPassword(),
+      });
+      cy.task("getEmployee").then((employeeInfo: any) => {
+        AddEmployeePage.fillFormAndSubmit(
+          employeeInfo.username,
+          employeeInfo.password
+        );
+      });
+      cy.url().should("contain", PersonalDetailsPage.path);
+    });
+
+    it("should find the new added employee in the employee list", () => {
+      DashBoardPage.clickElement(DashBoardPage.PIMTab);
+      PIMPage.clickElement(PIMPage.employeeListTab);
+      cy.task("getEmployeeId").then((employeeId: any) => {
```

Comment:
we should not have any type

Reply:
it makes errors when remove any or put any other type